home/blocks/button/_type/button_type_view-all.css

```diff
@@ -0,0 +1,7 @@
+.button_type_view-all {
+  margin-top: 27px;
+  color: #00a8cc;
+  cursor: pointer;
+  font-size: 16px;
+  line-height: 23.5px;
```

Comment:
shouldn't we use relative measurements instead of px for line height?

Reply:
we should